---
author: Meghana Reddy Dropathi
output:
  html_document:
    toc: true
    toc_depth: 4
    toc_float: true
    dev: 'svg'
  md_document:
    variant: gfm
bibliography: bibliography.ris
---
# Differential Expression Analysis

## Methods

In this project, we did differential expression analysis. We used Salmon[@Patro] to find all the Aip samples in the "/work/courses/BINF6309/AiptasiaMiSeq/fastq/ directory" and aligned them to AipIndex.In order to use the Salmon input in tximport[@Soneson] we created a table, mapping transcripts to genes. We have used annotation files in "/work/courses/BINF6309/data_BINF6309/Module4/Annotation/" and we showed the table using "kable"(part of knitr library). This creates a tx2gene.csv file which can be used further in analysis.We create a script called de.R which uses DESeq2[@Love] and tximport[@Soneson] to get the output. The script uses the DESeqDataSetFromTximport function to create a DESeqDataSet object from the tximport output. It specifies the experimental design with "Menthol" and "Vibrio" as factors. Rows  with counts less than 10 are removed to get better efficiency in the results as it removes low count genes. After this DESeq analysis is run.The dfAll.csv file is created. We filter the values of dfAll with padj less than 0.05 and we merge pathways,pathway names with results and write it into deAnnotated.csv. This csv file consists differentially expressed gene information with padj values less than 0.05.

## Results

```{R, code=readLines("de.R"), echo=TRUE, eval=TRUE,error=TRUE}
```
## References
